libs/libcommon/src/libcommon/parquet_utils.py

```diff
@@ -81,11 +81,16 @@ class ParquetIndexWithMetadata:
     httpfs: HTTPFileSystem
     hf_token: Optional[str]
 
+    num_total_bytes: int = field(init=False)
```

Comment:
When/where will we use this new attribute?

Reply:
it's not used, you're right. I was not sure if adding it, just in case we need it one day, or not. Better to remove it

Reply:
done